Source/WebCore/platform/audio/gstreamer/AudioDestinationGStreamer.cpp

```diff
@@ -161,21 +161,7 @@ AudioDestinationGStreamer::AudioDestinationGStreamer(AudioIOCallback& callback,
     // Link src pads from webkitAudioSrc to audioConvert ! audioResample ! [capsfilter !] queue ! autoaudiosink.
     gst_element_link_pads_full(m_src.get(), "src", audioConvert, "sink", GST_PAD_LINK_CHECK_NOTHING);
     gst_element_link_pads_full(audioConvert, "src", audioResample, "sink", GST_PAD_LINK_CHECK_NOTHING);
-
-    if (!webkitGstCheckVersion(1, 20, 4)) {
-        // Force audio conversion to 'interleaved' format (by audioconvert element).
```

Comment:
About this, cc @eocanha maybe we can keep this check, I'm not sure. Because at build time we would require 1.20.0, so that's a couple releases older than 1.20.4.

Reply:
The original requirement for gst 1.20.4 comes from [this downstream PR by @asurdej-comcast](https://github.com/WebPlatformForEmbedded/WPEWebKit/pull/1390/commits/09fb3f17d27fcc01d631abb07f057eb69a82de4f#diff-4e1e71d766bcfdc557c1d1edf1cf971fe05878660531fa909521bff643bb3de5R163), but I don't know exactly why that 1.20.4 version is important.